server/tserver/src/main/java/org/apache/accumulo/tserver/replication/AccumuloReplicaSystem.java

```diff
@@ -720,7 +719,7 @@ protected long writeValueAvoidingReplicationCycles(DataOutputStream out, LogFile
 
     // Add our name, and send it
     final String name = conf.get(Property.REPLICATION_NAME);
-    if (StringUtils.isBlank(name)) {
+    if (name.isBlank()) {
```

Comment:
I think this can be null.

Reply:
From my understanding, conf.get() should return the default if it doesn't find something set... which in this case its ""